### ISSUE TITLE
Code Security Audit 2&3

### ISSUE THREAD

**Issue by Ramu-kandimalla** [pucardotorg/dristi#1624]
- [x] Complete code deployment to Stable environment (UAT)
- [x] End to End flow testing 
- [x] identifying the Bugs 
- [x] Bugs Fixes and Sign-off
- [x] Code deployment to SDC UAT environment

**Comment by Pasam6** [comment]
Code is deployed to Azure Uat environment is completed by @Rkum113 
After bug fixes the same code was moved to SDC UAT environment in production network
SDC UAT is new environment (not the environment where we deployed resources for SA1)
**Deployment Issues:**
Issue 1:
Routes are to expose the service and gives a host name.
Not able to access the host name created even after adding host entry. 
nslookup is showing the ip address correctly 
curl is giving "(52) Empty reply from server" but I am able to access the pod using the clusterip of service after doing exec to any pod.
This issue was raised to SDC, SDC created a support ticket to openshift as they are facing the route issue for entire production network

Issue 2:
Resource Allocation of Memory and CPU. As the allocated resources are exhausted, we are facing latency issue which is impacting the performance
![Screenshot from 2024-09-10 18-42-59 (2)](https://github.com/user-attachments/assets/609ca670-c187-4a4b-822f-c274f688d8ca)
![Screenshot from 2024-09-10 18-40-57 (2)](https://github.com/user-attachments/assets/8a40d651-777b-4a9e-b270-2285cdc8d94e)

**Public Ip's**
As Advised by SDC we need to purchase public ip for internet access, currently we are working SDC to get this done, like from which isp we need to purchase and the process.


**Currently We are testing the application by port-forwarding the ingress service.**
